src/Native/Runtime/StackFrameIterator.cpp

```diff
@@ -458,6 +458,10 @@ void StackFrameIterator::InternalInit(Thread * pThreadToWalk, PTR_PAL_LIMITED_CO
     m_RegDisplay.pR9  = NULL;
     m_RegDisplay.pR10 = NULL;
     m_RegDisplay.pR11 = NULL;
+
+#elif defined(_TARGET_WASM_)
```

Comment:
It would be better to change this to `#elif defined(_TARGET_X86_) || defined(_TARGET_AMD64_)` and move the `PORTABILITY_ASSERT` into the `#else` block.

Reply:
Done